### PR TITLE
Layouts 637 new design

### PR DESCRIPTION
**Pull request by markosandalj**
Commented out "Add" button since it's out of scope for current release and ran the formatter. I also changed the html structure inside tests to match the new design but I'm not sure exactly how they work so that may still return errors. Here are tests from last commit https://github.com/netgen-layouts/layouts-ui-components/actions/runs/3395406062/jobs/5645246495

Task link: https://netgen.atlassian.net/browse/LAYOUTS-637

### CODE REVIEW

File: components/block/element.js [hunk 1]
```diff
@@ -233,14 +299,24 @@ export default class Block extends LitElement {
...
+      <main
+        class=${classMap(classes)}
+        @mouseover=${this.handleMouseover}
+        @mouseout=${this.handleMouseout}
+      >
         <div class="edit-menu">${this.renderMenu()}</div>
         ${this.renderBreadcrumbs()}
         <slot @click=${this.selectOnBlockClick}></slot>
-        ${this.renderAddButton()}
+        <!-- Out of scope for current release. Will be added in the next one ${this.renderAddButton()} -->
```
Comment (emodric) [inline components/block/element.js]: Can you use a separate commit to just remove this button, instead of commenting it out like this?

**Comment by emodric** [comment]
Thanks  @markosandalj ! I've left a minor comment for you.

As for failing tests, please consult with @bcafuk on why they are failing.

File: components/block/element.js (hunk 1 above)
Comment (markosandalj) [reply components/block/element.js]: Done. Here is commit https://github.com/netgen-layouts/layouts-ui-components/pull/2/commits/e7b4e33c4c4082002f9ff207225a3465ea0e881a

File: components/block/element.js [hunk 2]
```diff
@@ -53,28 +50,95 @@ export default class Block extends LitElement {
     return this.cached_parent;
   }
 
+  get parentName() {
+    if (!Boolean(this.parent)) return;
```
Comment (MarkoZabcic) [inline components/block/element.js]: You don't need `Boolean` here

File: components/block/element.js [hunk 3]
```diff
@@ -53,28 +50,95 @@ export default class Block extends LitElement {
...
+  get viewTypeName() {
+    return this.formatViewTypeName(this.model.attributes.definition_identifier);
+  }
+
   get isInLinkedZone() {
     return this.model.zone().is_linked();
   }
 
+  formatViewTypeName(name) {
+    if (!Boolean(name)) return;
```
Comment (MarkoZabcic) [inline components/block/element.js]: You don't need Boolean here

File: components/block/element.js [hunk 4]
```diff
@@ -84,21 +148,19 @@ export default class Block extends LitElement {
   }
 
   renderLinkedBlockMenu() {
-    return html`
-    <button class="refresh-btn" @click=${this.refresh}>
+    return html` <button class="refresh-btn" @click=${this.refresh}>
```
Comment (MarkoZabcic) [inline components/block/element.js]: Reformat it like this:
```js
return html`
  <button class="refresh-btn" @click=${this.refresh}>
    ${this.refreshIcon()}
  </button>
`;
```

Add new line after html` and before ` and prettier will reformat it like above

**Review by MarkoZabcic — CHANGES_REQUESTED** [review]

File: components/block/element.js (hunk 2 above)
Comment (markosandalj) [reply components/block/element.js]: Done https://github.com/netgen-layouts/layouts-ui-components/pull/2/commits/41b03365f396fa39af17bf90f4911f75540e1d01

File: components/block/element.js (hunk 3 above)
Comment (markosandalj) [reply components/block/element.js]: Done https://github.com/netgen-layouts/layouts-ui-components/pull/2/commits/41b03365f396fa39af17bf90f4911f75540e1d01

File: components/block/element.js (hunk 4 above)
Comment (markosandalj) [reply components/block/element.js]: Done https://github.com/netgen-layouts/layouts-ui-components/pull/2/commits/41b03365f396fa39af17bf90f4911f75540e1d01

**Comment by emodric** [comment]
Thanks @markosandalj !

**Pull request merged**